doc/source/autodoc.rst

```diff
@@ -72,3 +77,45 @@ Example
       portName3 : in bit    -- This is a documentation comment for portName3
     );
     end entity entityName;
+
+Example with groups
+-------------------
+
+Below shows how to work with port groups and generics.
+See :ref:`group_example_doc` for how this code will look
+
+.. code-block:: vhdl
+
+    entity entityName is
+    generic (
+        -- This is how the group of generics looks
+
+        -- =====================================================================
+        -- GROUP OF GENERICS
+        --
+        -- Description of group
+        -- =====================================================================
```

Comment:
Přemýšlím, jestli by nebylo lepší tuhle čáru nakonec přesunout mezi název a popis skupiny, co myslíš @martinspinler ?

Reply:
Pak by, ale nejlépe musela být ta čára i na konci description, aby bylo možné rozeznat, kde je description a kde pak výčet portů.

Reply:
Mně se to Danem navržené celkem líbí

Reply:
OK